app/models/visitor_supersession.rb

```diff
@@ -6,15 +6,24 @@ class VisitorSupersession < ActiveRecord::Base
 
   private
 
-  def merge_assignments! # rubocop:disable Metrics/AbcSize
+  def merge_assignments!
     target_split_ids = superseding_visitor.assignments.map(&:split_id).to_set
-    superseded_visitor.assignments.order(:id).map do |a|
+    superseded_visitor.assignments.order(:id).each do |a|
```

Comment:
i changed the map to each because we weren't doing anything with the results and this a private method that is called from a lifecycle hook.